services/core-api/app/api/utils/static_data.py

```diff
@@ -0,0 +1,50 @@
+from flask import current_app
+from sqlalchemy.inspection import inspect
+from sqlalchemy.dialects.postgresql import UUID
+
+from app.api.now_applications import models as app_models
+
+from app.api.constants import STATIC_DATA
+
+# This function is run right before setup_marshmallow and it looks through all of tables in our database.
+# It creates a mapping of classes to lists if their PK's, this is only done for code tables. To find the code tables
+# the classes are inspected and if they have a column named active_ind, it's type is not a UUID and its type is a string
+# the PK's are added to STATIC_DATA under the class name.
```

Comment:
Can you format this like a docstring """<>""" like in _deep_update_from_dict()? 

typo at 'lists **of** their PK's', don't need to list the conditions on what we are using to find a code, the code says that, maybe leave an inline comment that says 'code table conditions, to be refined later' or something like that.

Reply:
Changed to docstring style